CHANGELOG.md

```diff
@@ -8,7 +8,9 @@
 
 ##### Enhancements
 
-* None.
+* Add `RedundantNilCoalesingRule` Opt-In rule that warns against `?? nil`.
```

Comment:
Missing two trailing spaces, as noted in contributing.md

Also, this should be moved to a new top-level changelog section now that 0.12.0 is out.